internal/pkg/deploy/cloudformation/stack/app_test.go

```diff
@@ -404,6 +411,54 @@ Metadata:
 	require.Equal(t, AppResourcesConfig{
 		Accounts: []string{"0000000000"},
 		Version:  7,
-		Services: []AppResourcesService{{Name: "testsvc1"}, {Name: "testsvc2"}},
+		Workloads: []AppResourcesWorkload{
+			{Name: "testsvc1", WithECR: true},
+			{Name: "testsvc2", WithECR: true},
+		},
 	}, *config)
 }
+
+func TestAppResourcesService_UnmarshalYAML(t *testing.T) {
```

Comment:
Can we write these unit tests targeting the whole `AppResources`  instead of just the `Services` / `Workloads` fields?